### PR TITLE
[Accton/PDDF] Support pddf to as4630/as7816/as7326

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

#### Why I did it
Support pddf to as4630/as7816/as7326
#### How I did it
Send needed file to the PR for  these platform
#### How to verify it
Test sensors and show platform cmd.
root@as7326-56x-3:/home/admin# show platform psustatus
PSU    Model       Serial            HW Rev      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  ----------  ----------------  --------  -------------  -------------  -----------  --------  -----
PSU 1  FSF045-611  FSF0451912000505  N/A               12.06           5.50        66.00  OK        green
PSU 2  FSF045-611  FSF0451912000568  N/A               12.00           5.50        66.00  OK        green

root@as7326-56x-3:/home/admin# sensors
lm75-i2c-15-4a
Adapter: i2c-1-mux (chan_id 6)
Main Board Temperature:  +35.5 C  (high = +80.0 C, hyst = +75.0 C)

lm75-i2c-15-4b
Adapter: i2c-1-mux (chan_id 6)
CPU Board Temperature:  +29.0 C  (high = +80.0 C, hyst = +75.0 C)

fan_ctrl-i2c-11-66
Adapter: i2c-1-mux (chan_id 2)
fan1:        9100 RPM
fan2:        9400 RPM
fan3:        9300 RPM
fan4:        9600 RPM
fan5:        9000 RPM
fan6:        9100 RPM
fan7:        9100 RPM
fan8:        9300 RPM
fan9:        9200 RPM
fan10:       9400 RPM
fan11:       9200 RPM
fan12:       9400 RPM

pch_haswell-virtual-0
Adapter: Virtual device
temp1:        +43.0 C

psu_pmbus-i2c-17-59
Adapter: i2c-1-mux (chan_id 0)
in3:         +12.06 V
fan1:        6272 RPM
temp1:        +37.0 C
power2:       60.00 W
curr2:        +6.00 A

lm75-i2c-15-49
Adapter: i2c-1-mux (chan_id 6)
Main Board Temperature:  +40.0 C  (high = +80.0 C, hyst = +75.0 C)

lm75-i2c-15-48
Adapter: i2c-1-mux (chan_id 6)
Main Board Temperature:  +39.0 C  (high = +80.0 C, hyst = +75.0 C)

psu_pmbus-i2c-13-5b
Adapter: i2c-1-mux (chan_id 4)
in3:         +12.00 V
fan1:        6144 RPM
temp1:        +36.0 C
power2:       66.00 W
curr2:        +5.50 A

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +50.0 C  (high = +82.0 C, crit = +104.0 C)
Core 0:        +50.0 C  (high = +82.0 C, crit = +104.0 C)
Core 1:        +50.0 C  (high = +82.0 C, crit = +104.0 C)
Core 2:        +50.0 C  (high = +82.0 C, crit = +104.0 C)
Core 3:        +50.0 C  (high = +82.0 C, crit = +104.0 C)
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by FuzailBrcm — DISMISSED** [review]

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/pddf_post_driver_install.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ dis_i2c_ir3570a()
 {
     local addr=$1
 
-    i2cset -y 0 $addr 0xE5 0x01 &>/dev/null
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/pddf_post_driver_install.sh:7]: Can you also check the permission of this file, wether executable bit is set?

File: platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pddf_post_driver_install.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pddf_post_driver_install.sh:1]: Please check the permission of this file.

File: platform/broadcom/sonic-platform-modules-accton/as7816-64x/utils/pddf_post_driver_install.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as7816-64x/utils/pddf_post_driver_install.sh:1]: Please check the permission of the file

File: platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pddf_post_driver_install.sh (hunk 2 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pddf_post_driver_install.sh:1]: Done to update the permission of this file.

File: platform/broadcom/sonic-platform-modules-accton/as7816-64x/utils/pddf_post_driver_install.sh (hunk 3 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as7816-64x/utils/pddf_post_driver_install.sh:1]: Done to update the permission of this file.

**Review by FuzailBrcm — APPROVED** [review]
Changes are fine

**Pull request merged**